salt/daemons/flo/core.py

```diff
@@ -536,6 +536,9 @@ def _load_modules(self):
         self.modules.value = salt.loader.minion_mods(self.opts.value)
         self.returners.value = salt.loader.returners(self.opts.value, self.modules.value)
 
+        self.modules.value.clear()
```

Comment:
Actually, those 2 lines are now unnecessary. The original implementation of mine was singletons, but they aren't anymore. Meaning this will create an object and then immediately "clear" it, but it's already empty ;)